.github/workflows/merge-main.yml

```diff
@@ -61,6 +61,7 @@ jobs:
               -p URL=fom-test.nrs.gov.bc.ca -p KEYCLOAK_URL="https://test.oidc.gov.bc.ca/auth"
               -p FOM_EMAIL_NOTIFY=FLNR.AdminServicesCariboo@gov.bc.ca
               -p SITEMINDER_URL="https://logontest7.gov.bc.ca" -p DB_TESTDATA=true
```

Comment:
I assume the siteminder_URL is no longer needed for logout and can be removed.

Reply:
A ticket to clean up is at the backlog. Intended to remove it altogether.